new/chef-workstation.md

```diff
@@ -0,0 +1,108 @@
+RFC: unassigned
+Title: The Future of Chef Workstation
+Author:
+- Jon Morrow <jmorrow@chef.io>
+- Noah Kantrowitz <noah@coderanger.net>
+Status: Draft
+Type: Informational
+---
+
+# The Future of Chef Workstation
+
+ChefConf 2018 saw the unveiling of both the `chef-run` command and a Chef Workstation
+installer package containing it. This has lead to user confusion on two fronts.
+First, the term "Chef run" (or "Chef client run") is already in very common use
+in our community, so trying to communicate about `chef-run` has been tricky.
+Second, having two installers (Chef Workstation and ChefDK) has resulted in a
+lack of certainty about which should be used.
+
+## Motivation
+
+    As a Chef user,
+    I want to use the new Chef functionality,
+    so that I can manage machines/resources without Chef Server.
+
+    As a Chef community member,
+    I want to communicate with other Chef community members,
+    so that I can share knowledge and answer questions.
+
+## Specification
+
+1. The `chef-run` command will be renamed to `chef-target` [ed: not attached to this name, suggest a better on in comments].
+2. The `chef-target` gem will be split into its own repository for future development.
+3. The installer currently called "ChefDK" will be renamed "Chef Workstation".
+   This will require setting up documentation to explain the rename, as well as
+   redirects and supporting pages in the download system to ensure users don't
+   get lost.
+4. The `chef-dk` gem will not be renamed at this time, as few users see the gem
+   name there and we're already changing a lot of things. This may be revisited
+   in the future when the dust settles.
+5. The installer for `chef-dk` (now named "Chef Workstation") will be reconfigured
+   to install into `/opt/chef-workstation` and use `~/.chef-workstation` as the
+   primary configuration folder. A symlink for `/opt/chef-dk` (and related windows-y paths)
+   should be added to ensure any scripts that hardcode paths like `/opt/chefdk/bin/chef`
+   continue to function (until such time as we decide to remove them as compatbility layers).
+   Gems in `~/.chefdk/gems` should be added to the gems path, and config files in
+   `~/.chefdk` should work if no file in `~/.chef-workstation` takes priority.
+6. Add the `chef-target` gem to the Chef Workstation installer.
+7. A command stub will be added to the `chef-dk` command processor called `target` [ed: or whatever we call it],
+   which will dispatch to the `chef-target` command. The Chef Workstation installer
+   will map `chef-target` into `embedded/bin/` rather than `bin/` so for most users,
+   this will be the UX.
+
+Additionally, while it is out of scope for this RFC, the authors strongly encourage
+future RFCs and discussions about adding additional tools to Chef Workstation/DK
+that are of value to the Chef community.
+
+## Alternatives
+
+### Keeping the "ChefDK" Name
+
+While "Chef Workstation" is a clearer name, there is broad community usage and
```

Comment:
I think we could potentially call out what "clearer" means - if we're on board with my previous comment about `Chef Workstation` being focused on end-user desktops and people who need to manage their Chef infrastructure from their laptop, we should be explicit about that.